components/director/internal/domain/tenant/resolver.go

```diff
@@ -393,6 +400,18 @@ func (r *Resolver) fetchTenant(ctx context.Context, tx persistence.PersistenceTx
 	return tr, nil
 }
 
+func (r *Resolver) syncSystemsForTenant(ctx context.Context, tenantID string) {
+	if err := r.systemFetcherClient.Sync(ctx, tenantID); err != nil {
```

Comment:
Here can be added a log also.

Reply:
Not needed additional logs, it uses a function that logs properly.